mappings/net/minecraft/server/command/CommandOutput.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_2165 net/minecraft/server/command/CommandOutput
+	COMMENT Represents a subject which can receive command feedback. 
 	FIELD field_17395 DUMMY Lnet/minecraft/class_2165;
-	METHOD method_9200 sendCommandFeedback ()Z
+	METHOD method_9200 shouldSendCommandFeedback ()Z
```

Comment:
```suggestion
	METHOD method_9200 shouldReceiveFeedback ()Z
```
This is already a "command output", and this output is not sending feedback but receiving feedback